src/explorer/SiteTreeItem.ts

```diff
@@ -167,6 +178,22 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
     }
+
+    private async promptToSaveDeployConfigs(node: IAzureNode<SiteTreeItem>, activeWorkspacePath: string, deployedProjectPath: string): Promise<void> {
```

Comment:
Worth passing the reporter to get some more telemetry on this?

Reply:
You shouldn't have to pass the reporter! :) That's the whole point of this work: https://github.com/Microsoft/vscode-azureappservice/pull/495